modin/experimental/cloud/local_cluster.py

```diff
@@ -57,9 +61,9 @@ def _run(self, sshcmd: list, cmd: list, capture_out: bool = True):
             redirect.write(f"Running: {cmd}\n")
         return subprocess.Popen(
             cmd,
-            stdin=subprocess.DEVNULL,
-            stdout=redirect,
-            stderr=redirect,
+            stdin=None if _IS_PYTEST_DEBUG and not capture_out else subprocess.DEVNULL,
```

Comment:
I'm not familiar with these test fixtures, so could you explain these lines? If you're running `pdb` (and `not capture_out`) why should `stdin`, `stdout`, and `stderr` be None?

Reply:
These are not fixtures, it's a normal `subprocess.Popen()` being called here.
And the reason for those to be `None` is so the "remote" end actually can be interacted with using std streams.